src/main/java/com/dumbdogdiner/stickyapi/common/util/TextUtil.java

```diff
@@ -4,12 +4,13 @@
  */
```

Comment:
Worth merging this class with `StringUtil` since they both deal with text? I understand that one is more closely related to characters than entire strings, but seems like a sensible idea to me.